packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -362,6 +437,19 @@ class SkiaGoldClient {
     }
   }
 
+  /// Returns the number of commits made on the current branch.
+  Future<String> getBranchCommitCount() async {
+    final String branch = platform.environment['CIRRUS_BRANCH'];
+    final String masterBranch = platform.environment['CIRRUS_BASE_BRANCH'];
+
+    // This git wizardry does not appear to work as intended on Cirrus.
+    final io.ProcessResult revList = await process.run(
+      <String>['git', 'rev-list', branch, '^$masterBranch', '--count'],
+      workingDirectory: _flutterRoot.path,
+    );
```

Comment:
@kjlubick, getting the nth commit like I planned to doesn't work on Cirrus, I'm looking into another way to get the number.

Reply:
That's fine; To test things out you could hard-code a number (e.g. 1) for now